assembly/debug/debug_mode.s

```diff
@@ -0,0 +1,1380 @@
+.thumb
+.align 2
+
+.include "../xse_commands.s"
+.include "../xse_defines.s"
+.include "../asm_defines.s"
+
+.global EventScript_DebugMode_Main
+EventScript_DebugMode_Main:
+	givepokemon SPECIES_BULBASAUR 0x32 0x0 0x0 0x0 0x0 @lv 50 Bulbasaur
+	givepokemon SPECIES_CHARMANDER 0x32 0x0 0x0 0x0 0x0 @lv 50 Charmander
+	givepokemon SPECIES_SQUIRTLE 0x32 0x0 0x0 0x0 0x0 @lv 50 Squirtle
+	givepokemon SPECIES_GROOKEY 0x32 0x0 0x0 0x0 0x0 @lv 50 Grookey
+	givepokemon SPECIES_SCORBUNNY 0x32 0x0 0x0 0x0 0x0 @lv 50 Scorbunny
+	givepokemon SPECIES_SOBBLE 0x32 0x0 0x0 0x0 0x0 @lv 50 Sobble
+
+	@mark all pokemon in the dex as seen
```

Comment:
add a space after the @ to be consistent with the other comments

Reply:
Fixed